### DISCUSSION TITLE
disable counter

### DISCUSSION THREAD

**Discussion by dryodryo** [audioscavenger/save-image-extended-comfyui#20] [Ideas]
Hi, 

Thank you for an incredibly useful custom node!

I would love to be able to disable the counter entirely. I'm already adding a huge amount of info to the filename, and don't need the counter except for image sequences.

Thanks again

**Comment by audioscavenger** [comment]
sure, that's 2 of you willing an option to disable the counter, i take note

**Comment by TheBitcoinBaron** [comment]
I also would like an option for this... https://github.com/thedyze/save-image-extended-comfyui/issues/27

**Comment by audioscavenger** [comment]
counter can now be disabled using 0 digits: 
![image](https://github.com/user-attachments/assets/9b6d3c0f-92df-4b93-ac7e-0803bff99efc)
